galata/src/helpers/notebook.ts

```diff
@@ -883,7 +883,7 @@ export class NotebookHelper {
     if (gutters.length < lineNumber) {
       return false;
     }
-    await gutters[lineNumber - 1].click();
+    await gutters[lineNumber].click();
```

Comment:
I guess it was an attempt at tackling https://github.com/jupyterlab/jupyterlab/issues/15008?

Reply:
Yes, but it was incorrect and I have reverted it.

Reply:
I think I have fallen for thinking this would be a fix before too. Just wanted to share the link to the issue tracking it for reference :)